book_haven_api/tests/requests/libraries/books/test_add_book_to_library.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+from django.urls import reverse
+from rest_framework.test import APIClient
+from book_haven_api.models import Library, Book
+
+@pytest.fixture
+def Libraries():
```

Comment:
The convention here would be to use a snake case, and I'd also switch to singular since it's just returning a single Library. Something like:
```suggestion
def single_library() -> Library:
```
This also adds in a return type hint, which isn't required but is definitely helpful.

Reply:
Ah yes, we didn't realize the plurality. Thanks for pointing that out!